app/components/cards/Comment.jsx

```diff
@@ -48,8 +47,9 @@ export function sortComments( g, comments, sort_order ) {
                 return bactive.getTime() - aactive.getTime();
               },
       new:  (a,b) =>  {
-                let acontent = g.get('content').get(a);
-                let bcontent = g.get('content').get(b);
+                let acontent = cont.get(a);
+                console.log("acontent", acontent);
```

Comment:
Hm that's weird that was definitely removed in my local code, I'll see if I can add a commit for that.